File: bubblesort.c
#include<stdio.h>
#include<stdlib.h>
#include<time.h>
void bubbleSort(int A[],int n)
{
    int i,j,temp;
    for(i=0;i<n-1;i++)
    {
        for(j=0;j<n-i-1;j++)
        {
            if (A[j] > A[j+1])
            {
                temp=A[j];
                A[j]=A[j+1];
                A[j+1]=temp;
            }

        }
    }
}
void display(int A[],int n)
{
    int i;
    for(i=0;i<n;i++)
    {
        printf("%d ",A[i]);
    }
    printf("\n");
}
int main()
{
    int A[100000],n,i;
    clock_t start,end,diff;
    printf("Enter n:");
    scanf("%d",&n);
    for(i=0;i<n;i++)
    {
        A[i]=rand();
    }
    display(A,n);
    start =clock();
    bubbleSort(A,n);
    end=clock();
    display(A,n);
    diff=end-start;
    printf("Time Taken is %f second\n",(float)diff/CLOCKS_PER_SEC);
    return 0;
}

File: bubblesort.py
import time
import random

def bubbleSort(A,n):
    for i in range(0,n-1):
        for j in range(0,n-1-i):
            if A[j]>A[j+1]:
                temp=A[j]
                A[j]=A[j+1]
                A[j+1]=temp

def display(A,n):
    for i in range(0,n):
        print(A[i],end=" ")
    print(" ")

A=[0]*100000
print("Enter n:",end=" ")
n=int(input())

for i in range(0,n):
    A[i]=random.randint(0,10)
display(A,n)
start=time.time()
bubbleSort(A,n)
end=time.time()
display(A,n)
diff=end-start
print(f"The time taken is {diff:.6f} seconds")